src/main/kotlin/no/nav/syfo/JacksonKafkaSerializer.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.syfo
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import org.apache.kafka.common.serialization.Serializer
+
+class JacksonKafkaSerializer : Serializer<Any> {
+    private val objectMapper: ObjectMapper = ObjectMapper()
```

Comment:
du har allerede en definert ObjectMapper i Appliction State. Går det an å bruke den?

Reply:
Jeg er litt usikker på om det er så lurt å bruke samme mapperen til å lese config som vi bruker i serializeren.. Enten må jeg definere mapperen utenfor og sendte den inn til serializeren som constructor-arg, men jeg må fortsatt ha den configure-metoden der, for den kalles potensielt utenfor min kontroll.. Eller så kan jeg definere mapperen kun i serializeren, men da må jeg ha en get på den for å hente den ut for å lese configfilene.. Ingen av delene føles helt bra, men jeg skjønner poenget med å definere to mappere.. 

Eller er det andre måter å gjøre det på..? :)